index.html

```diff
@@ -10,10 +10,53 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
-
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
-
+    <header id="header">
+        <img id="logo" src="https://svgshare.com/i/shA.svg" alt="Karma Logo" width="50px" height="40px"/>
+        <nav id="navbar">
+            <ul>
+                <li><a id="dark-link" href="#">Meet Karma</a></li>
```

Comment:
`id="dark-link"` along with the separate css rule that element could be replaced with `nth-child(1)` selector in css:
```css
nav li:nth-child(1) a {
    color: black;
    text-decoration: none;
}
```
see more about it here: [MDN - CSS - nth-child](https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child)